docs/mongocli/command/mongocli-cloud-manager-logs-decrypt.txt

```diff
@@ -83,5 +83,6 @@ Examples
 .. code-block::
 
 
-   $ mongocli ops-manager logs decrypt --localKeyFile filePath --file logPath --out resultPath
+   $ mongocli ops-manager logs decrypt --localKeyFile /path/to/keyFile --file /path/to/logFile.bson --out /path/to/file.json
```

Comment:
[q] Why the duplicated line?

Reply:
Not duplicated, different example json and bson log files